mern-ecommerce/routes/user.js

```diff
@@ -17,8 +17,8 @@ router.get('/secret/:userId', requireSignin, isAuth, isAdmin, (req, res) => {
 });
 
 router.get('/user/:userId', requireSignin, isAuth, read);
-router.put('/user/:userId', requireSignin, isAuth, update);
-router.get('/orders/by/user/:userId', requireSignin, isAuth, purchaseHistory);
+router.put('/user/:userId', requireSignin, isAuth, updatee);
```

Comment:
There is a typo in the code. The function name is `update` but it's called as `updatee`. It should be corrected to `update`.

Reply:
@CodeHawk Can you please further clarify on your review in simple terms?

Reply:
You've got a small typo in the code. The function name is `update`, but you're calling it as `updatee`. It should be corrected to `update`.